components/search/index.css

```diff
@@ -226,14 +210,26 @@
 
 /* Quiet Variant */
 .spectrum-Search--quiet {
-	.spectrum-Search-clearButton .spectrum-ClearButton-icon {
-		transform: translateX(var(--mod-search-quiet-button-offset, var(--spectrum-search-quiet-button-offset)));
+	--spectrum-search-background-color: transparent;
+	--spectrum-search-background-color-disabled: transparent;
+	--spectrum-search-border-color-disabled: var(--spectrum-disabled-border-color);
+
+	/* Added specificity, otherwise they are overridden by system specific themes. */
+	&.spectrum-Search {
+		--spectrum-search-border-radius: 0;
+		--spectrum-search-edge-to-visual: var(--spectrum-field-edge-to-visual-quiet);
 	}
 
 	&.spectrum-Search .spectrum-Search-input {
 		border-radius: var(--mod-search-border-radius, var(--spectrum-search-border-radius));
 		padding-inline-start: calc(var(--mod-search-edge-to-visual, var(--spectrum-search-edge-to-visual)) + var(--mod-search-icon-size, var(--spectrum-search-icon-size)) + var(--mod-search-text-to-icon, var(--spectrum-search-text-to-icon)));
-		padding-inline-end: calc(var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size)) - var(--mod-search-quiet-button-offset, var(--spectrum-search-quiet-button-offset)));
+		padding-inline-end: calc(var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size)) - var(--mod-search-quiet-button-offset, calc(var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2 - var(--mod-search-icon-size, var(--spectrum-search-icon-size)) / 2)));
 		padding-block-start: var(--mod-search-top-to-text, var(--spectrum-search-top-to-text));
 	}
+
+	.spectrum-Search-clearButton .spectrum-ClearButton-icon {
+		/* @deprecated --mod-workflow-icon-size-100 -- use --mod-clear-button-width instead */
+		/* stylelint-disable-next-line spectrum-tools/no-unknown-custom-properties -- variable --spectrum-clear-button-width will resolve via the .spectrum-ClearButton class */
+		transform: translateX(var(--mod-search-quiet-button-offset, calc(var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2 - var(--mod-clear-button-width, var(--mod-workflow-icon-size-100, var(--spectrum-clear-button-width))) / 2)));
+	}
```

Comment:
Is this `transform` still needed? I disabled this style in the inspector for all t-shirt sizes and I'm not seeing any visual changes. It looks like the calc() is calculating to 0.  I see the icon is centered within the button with the button fill element's flex properties.

Reply:
Great point, I've pushed up this update.

Reply:
That mod is also being used for the inline-end which is more relevant to placement so it should still do what customers are expecting.